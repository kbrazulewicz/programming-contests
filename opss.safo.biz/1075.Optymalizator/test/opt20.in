10
500 -186610
75 -447 118 -384 479 239 78 398 -7 -387 -477 309 462 -251 -230 426 207 -315 93 -472 -408 228 101 -141 441 -307 -355 -110 -253 -195 280 322 -142 250 -209 190 341 369 88 -165 -166 463 -4 -204 -288 -382 75 -228 155 20 -200 -400 -251 401 311 42 -53 457 -215 194 -238 -83 368 472 20 -341 162 -139 -120 102 -452 214 66 396 -137 -370 -485 438 -98 170 310 -445 270 -89 456 82 454 255 391 91 -51 5 -492 -182 -23 28 329 -8 -258 -290 -406 -210 -224 -488 -313 139 -357 54 429 397 -276 240 452 -153 -349 -239 -219 -43 -484 -328 -452 -34 177 -91 -364 6 437 465 498 31 27 -55 322 304 -43 -139 295 452 415 -423 350 -9 -331 -346 338 172 -85 -29 -370 -68 143 30 398 172 439 -114 -321 -271 -149 29 260 -269 474 434 -113 284 147 34 -412 62 111 -210 -94 280 -55 -256 305 360 -432 435 -208 -437 317 42 236 257 428 -233 -162 132 -352 -50 363 -25 237 -250 -241 -116 284 -153 447 248 138 -147 380 -65 449 185 -205 -131 472 88 -67 290 -370 21 399 -89 288 -263 -457 -212 -461 -242 263 276 8 -126 -339 144 222 -40 -108 212 313 125 -353 -386 -190 294 -16 135 -118 -231 -75 365 290 -176 128 430 413 171 -282 452 429 482 -419 289 -144 -258 285 -70 202 30 -6 367 -345 493 -167 317 288 -331 -48 22 -62 -271 239 -272 405 -133 -342 318 -110 229 -377 -181 -437 -296 -40 271 298 -254 -298 352 276 -452 -429 -217 -106 -96 -400 34 74 405 -444 -488 -366 -352 93 -460 15 -397 -290 406 332 185 77 -253 -259 -110 -481 -461 136 -279 391 264 121 -186 -453 -485 219 -1 401 145 256 458 -491 391 -42 102 -217 473 206 345 -269 390 31 -339 -362 -376 51 9 -336 39 230 -93 -197 203 222 202 -429 293 53 472 -62 -190 -218 -53 53 240 -450 -312 -434 -392 -467 149 -150 416 310 340 -459 213 -151 57 -248 -69 -184 407 -365 -462 461 -294 331 15 30 121 -323 313 -431 230 53 -29 -230 119 431 -197 121 -219 72 -69 -26 465 -355 -325 22 249 107 338 -491 -406 -271 470 152 -88 337 -318 -466 -486 495 -45 -404 401 426 -134 372 357 170 -7 -10 94 425 464 59 -78 -360 -419 23 -401 -229 32 -307 0 355 -303 413 192 -121 -201 -441 227 254 -345 128 180 22 0 389 44 -154 -121 -362 123 196 -451 -455 -312 -18 420 139 253 305 -316 106 160 -119 371 204 -388 -330 263 -309 -224 419 -329 456 293 -476 -303 -163 -130 
500 90928
327 493 476 376 -110 -336 210 -190 -345 -37 -33 -161 -79 -21 -428 -208 -316 -315 314 -201 228 -410 70 400 -102 -137 424 95 -448 -206 376 379 -213 352 -393 -471 -131 317 191 24 133 -341 364 -446 138 436 199 322 -27 365 -26 -298 456 -456 -46 206 260 -122 154 312 172 30 192 -189 -118 -349 192 251 469 -117 -224 454 42 -8 -492 -319 280 -441 -145 106 425 329 -340 -267 -274 114 -209 486 492 445 298 -484 327 342 179 -290 494 -129 -187 -185 254 441 -231 -203 285 129 330 66 -459 37 -328 466 -133 184 51 445 -202 342 431 -210 -360 -419 158 467 424 -163 29 418 -440 -305 85 314 -364 354 -37 422 335 145 340 376 -317 -136 -306 402 -452 245 347 -154 440 278 488 -68 359 498 251 -365 -165 -219 -95 247 -24 490 61 464 -304 -123 386 -468 -478 -422 408 -295 -58 -397 -393 342 -300 454 -312 -8 -416 28 424 -204 26 176 -69 -287 -191 337 460 285 -321 374 249 -124 -249 488 -92 125 -82 -184 331 361 271 -210 203 472 -255 -256 -36 181 -376 241 477 -497 -83 261 68 226 450 29 363 -371 255 464 5 -142 -196 -235 483 223 434 -334 436 205 -43 491 -471 -446 87 -6 235 212 87 -435 -433 -496 326 135 -418 276 16 -203 -243 -229 113 -385 129 418 -120 465 493 -186 -369 429 -128 440 420 -99 494 8 -253 230 72 334 -353 139 -310 473 -226 124 -399 291 421 -142 -86 387 -27 -104 305 -294 213 298 20 344 227 -256 -363 499 498 -369 359 97 -287 -69 284 -140 -78 -174 185 197 451 286 340 -276 -3 106 -389 470 -498 -232 -324 215 -434 -451 412 -355 -207 -451 -3 143 32 -144 -259 -402 -360 -475 -42 62 -149 -4 -241 154 -218 -49 379 279 -442 -158 -398 -88 111 -222 -372 -471 179 -108 27 -175 -59 -476 468 -175 -120 -291 -77 20 86 -266 -417 -210 230 194 444 -488 -2 -325 -356 56 370 -254 -180 -19 376 -52 362 56 340 -111 381 133 -87 349 -189 294 -89 234 166 -151 468 -399 139 -450 -352 436 -437 146 -389 59 54 481 -343 374 314 -467 175 177 441 367 -434 322 353 -20 -476 -336 126 -65 -250 144 -364 -281 -402 276 -231 -254 212 184 -256 -325 95 298 -343 252 -476 -177 -362 199 -500 -421 419 -81 402 272 399 -222 436 377 213 -314 373 -299 257 471 477 -121 -283 -459 -85 -39 -283 -489 111 -126 263 136 -451 401 187 -450 -167 106 -31 -265 -122 220 13 -186 449 78 353 322 279 462 294 -391 -159 -489 -350 257 
500 -32175
-133 268 -64 -407 383 -428 -357 -363 -388 45 -30 218 -134 -443 -403 -62 422 263 -113 0 468 61 131 431 -145 240 -228 219 -257 -119 44 110 -351 -168 -444 -115 257 -301 374 369 -404 -156 87 -38 253 184 400 175 300 139 27 268 -300 -342 -449 408 -249 -176 127 346 205 23 -44 -293 355 12 444 112 -437 318 -19 -341 -338 -431 473 -233 -395 -275 442 -243 364 -31 -122 -83 479 -71 -175 82 253 452 -72 311 -25 -263 -130 182 249 -186 -205 -335 -368 276 176 146 -303 150 413 155 375 355 412 -408 176 -210 9 155 220 -314 -262 325 -10 -482 136 -183 -245 -494 499 -143 172 -206 22 304 -77 -302 302 472 -300 215 127 -72 422 40 20 98 -318 -119 106 402 419 344 -420 -91 362 216 226 -382 -425 225 -25 -401 -128 -151 404 -353 399 206 -381 -400 274 -401 28 -452 139 -100 147 -327 -367 -247 -72 -448 -51 8 -39 163 76 39 -367 -497 116 460 -397 -160 -191 -493 -13 -291 -435 459 161 -309 -442 41 -260 -451 441 -261 -278 -74 344 -498 -22 -207 10 -209 -192 -413 -318 -58 -58 298 254 45 -9 -436 52 478 125 470 -211 138 13 -153 -321 -395 248 -28 -156 -177 398 -312 -175 228 333 -312 -129 -358 127 53 436 -431 203 190 115 -306 -394 19 25 -417 341 314 -427 -145 14 -248 -40 114 -424 -343 437 -174 -155 115 -446 31 303 -75 -475 -70 -170 461 351 -466 -497 -34 80 -38 486 -395 397 327 420 471 34 286 -277 -153 -100 -200 4 190 478 -299 -195 -467 232 -40 310 257 -258 -7 -430 -407 379 -74 412 -41 388 398 65 285 -423 337 256 464 475 -168 -189 -125 484 315 -83 462 -132 -426 -153 -399 34 -342 -290 -372 -497 133 -278 382 -441 -366 -159 -201 -116 406 -64 313 -405 -307 277 -430 377 -412 298 361 -245 -285 -325 476 -210 23 -423 -324 -467 139 -195 -464 -228 27 418 -169 -487 -241 482 249 18 -81 62 113 464 -160 184 341 280 -18 202 -112 -451 -271 -136 -161 252 293 16 285 -68 321 321 205 200 239 -112 -435 -149 223 314 369 -358 -124 -18 -394 -432 18 -201 349 -148 -147 89 -98 82 453 241 -165 -402 109 -380 30 282 -206 87 334 -115 -172 399 236 -449 -435 457 45 294 292 -497 362 -190 -198 -437 -485 155 152 -83 -263 457 10 72 -93 -380 -455 438 -246 191 377 -411 -424 57 -160 -335 460 258 -26 5 52 -234 360 -86 429 -338 330 444 317 334 213 -445 -356 -277 479 51 195 -124 -11 450 -433 219 -109 -4 -224 
500 -1917
-487 -263 -307 -13 -406 245 -394 455 12 -113 -383 -158 -169 -213 -324 -104 194 -180 119 173 372 167 -98 213 469 321 -68 -140 169 -439 295 182 150 489 22 244 -414 480 199 98 -133 -331 -208 -450 -44 321 -54 -350 -359 -83 -325 -135 -416 429 -69 -447 251 -285 265 -80 -224 61 -45 426 402 -171 -477 488 161 -426 439 -472 -257 -269 -70 -449 -448 376 -299 46 145 229 -237 230 158 194 283 261 -238 49 -318 390 -38 489 -331 -136 170 -308 204 -169 -234 143 211 362 -273 -359 413 -221 -131 -33 325 -486 -304 -411 244 -294 -365 380 468 249 281 2 140 -257 343 309 459 13 -147 163 344 119 159 55 -19 386 -452 247 17 -83 -286 -157 283 262 284 380 -180 -229 260 -212 -479 41 -358 161 284 485 322 -405 350 -325 -390 194 -354 -379 101 480 -493 149 227 377 -82 441 -280 -298 -445 -144 -418 -125 479 -306 -484 -500 235 -342 -487 371 -4 335 -182 -154 -138 -72 -107 9 50 494 -11 409 496 -284 286 266 9 -142 -180 416 -434 254 -209 -454 448 -193 -454 -465 318 -88 -242 166 99 76 -488 462 -143 257 323 -241 -396 164 -332 -400 380 -193 -282 241 165 39 157 -416 -207 -200 -370 94 460 28 129 278 440 388 296 40 316 308 354 -327 418 177 -68 -478 -159 453 -26 73 -240 192 166 -223 -269 175 -139 377 475 -9 -29 -213 -480 -48 -83 -188 -160 -287 352 9 -126 -442 182 -208 235 -33 -186 -72 -80 288 -147 180 -20 -481 -191 -436 194 171 -207 -478 14 264 -191 386 68 227 -301 409 440 -97 -82 -186 -38 -48 106 49 419 420 478 -161 -440 -169 -129 41 203 181 457 -103 204 102 271 70 -134 -67 -43 -214 -340 156 -305 -48 -441 465 267 373 -82 225 -77 189 498 -99 -471 -442 -416 252 451 287 285 -240 37 -11 362 -192 -440 80 241 369 367 -247 25 -86 206 436 -120 325 310 -350 402 85 339 400 486 -280 -189 -78 473 114 -290 -390 375 247 100 -411 -93 -488 170 1 381 389 -394 258 -197 312 -454 -465 489 -292 -315 392 293 -123 -356 -369 449 -45 54 -226 -78 264 -115 149 363 485 238 122 -151 -92 -25 -270 -203 82 -160 453 -106 -114 488 236 447 26 128 240 403 -376 372 352 432 278 -373 -146 394 364 -497 -391 349 241 231 50 2 -293 280 -349 289 -28 -396 35 358 -407 123 -343 471 -249 398 374 -124 -378 -422 -192 400 57 -486 -354 -79 -483 -245 122 -390 486 -328 112 45 -196 264 186 276 -280 74 487 165 -303 
500 -212856
136 301 -106 10 29 16 441 -311 268 -2 -297 414 -228 -280 -331 394 182 8 -81 -205 -95 223 -89 -56 0 131 -482 339 -203 67 483 -215 -132 -122 296 -251 -254 -411 -62 -485 439 141 -219 -289 -287 303 -42 396 163 377 43 -432 100 454 12 -48 85 382 -209 -266 302 127 20 -330 -495 168 272 -249 257 -290 118 -452 -296 400 112 -83 203 -430 165 -134 -201 -292 286 -249 -338 151 204 -400 33 495 -166 187 122 -294 358 479 374 482 83 483 -308 -299 -116 -104 -399 496 -334 -344 -82 331 374 217 -460 161 468 54 -188 -476 -494 -303 -128 193 385 -154 399 -405 -174 -374 -423 409 -39 121 -38 345 -482 416 -307 -316 -428 111 -133 447 180 259 460 149 -186 124 173 172 -327 -455 -135 -90 -108 -235 357 -430 243 -214 -169 -296 408 293 -98 426 209 447 -38 -366 59 329 -419 -261 441 -107 -260 107 -131 -234 -221 42 163 145 -195 -93 410 -338 -23 -495 -51 -340 61 209 454 463 487 163 -237 -51 297 322 130 -270 -87 423 475 -494 30 -156 -228 310 -261 435 -45 396 195 217 -90 24 222 359 185 -217 420 139 99 407 154 362 208 -48 -464 191 -466 301 114 -138 307 -355 -442 -69 455 -203 219 -90 45 -86 -373 456 438 -299 -333 123 336 -412 -238 -65 495 -231 149 204 -427 -463 -105 -393 339 -491 321 498 -346 380 -70 -391 29 -351 371 -425 415 498 -469 353 51 50 329 -112 138 -57 -325 134 212 -323 190 285 -286 -63 -255 -95 -54 -434 256 453 446 -314 -86 328 187 -214 403 454 136 -214 307 -312 336 488 428 -25 -68 -397 -39 -356 -220 3 -218 347 440 27 104 238 445 -140 191 392 398 -42 220 437 244 475 391 232 -239 -449 -228 -51 -461 -300 -224 323 -196 -411 -180 84 92 102 283 32 -19 -112 -229 -74 248 -186 170 499 272 242 436 368 217 328 101 -170 -269 373 280 -230 -426 -92 -54 230 -2 266 166 -410 -132 -50 475 349 338 -254 127 438 412 -202 437 185 -460 -274 53 258 -94 6 -60 137 232 72 -241 -342 481 205 388 -21 -29 54 421 -309 -144 396 -108 46 -6 -480 337 -93 -182 -374 -56 -142 -148 -151 -32 258 -292 409 -105 440 -19 155 -402 314 -140 -162 145 -316 -256 67 -125 101 315 268 -1 162 288 -164 -79 -42 -37 365 316 -333 66 137 426 274 46 173 214 -121 180 -336 46 -459 2 -309 77 -401 -390 452 200 426 -428 51 440 360 388 -139 -182 203 -274 -13 370 -356 -376 -352 419 -478 322 485 
500 -21247
2 2 151 395 -496 343 -28 -397 -195 277 -345 83 349 207 23 -438 447 384 232 150 -38 71 372 107 -453 -479 378 -79 -305 363 -325 -451 -135 326 445 222 -479 269 -323 -173 398 185 410 -400 244 -66 14 191 170 98 193 -367 -330 417 -408 -283 -210 470 139 -15 -315 -334 387 403 -156 -316 125 -134 453 -346 -455 -296 339 -193 156 83 -407 -330 126 264 268 171 -251 -210 89 -159 8 379 -189 -1 217 -4 -483 456 -101 -139 -360 -124 -421 -55 31 -24 149 222 284 -195 158 377 327 136 141 96 -192 390 -262 -251 83 -402 -20 394 97 49 243 466 5 -358 -172 145 371 259 -409 -98 -264 -408 -24 -480 -102 -14 397 77 -377 391 25 -69 -367 264 -468 217 362 12 463 312 -438 206 278 67 -299 -394 -287 -428 366 156 326 -46 248 -198 -26 -2 289 223 -424 -236 466 101 195 -48 -135 227 21 -272 91 -16 -108 153 43 -330 73 244 -223 -362 316 -5 294 494 449 394 148 275 -107 -63 -2 -31 -447 -35 -430 -400 417 288 327 -62 -132 271 -226 260 276 317 430 -151 61 59 339 229 -94 133 223 207 380 -128 -18 -227 -339 -167 242 67 298 -336 167 -433 452 347 357 172 118 -17 -68 -106 -199 215 -404 -286 -226 435 -204 181 421 -129 240 301 -405 -277 -426 109 56 -332 -324 206 -168 343 -375 137 190 -166 161 -192 317 -406 -445 -30 -339 -349 -463 -65 -62 -167 -32 -141 204 -291 -340 300 284 86 -91 -160 -394 85 -102 291 280 23 280 -29 -291 -59 -369 -122 387 -314 349 -452 -163 386 -16 276 -281 -196 -13 -225 365 -500 -425 149 438 336 -159 -455 273 239 336 54 114 116 -475 323 -443 -344 54 445 195 -97 -155 -116 141 329 -340 -288 -14 -500 -13 -149 -500 415 -499 438 -397 -158 -165 377 -66 -329 283 -100 287 -340 -276 -303 -184 -370 -6 -137 33 339 248 26 -479 -240 238 7 -240 77 -290 112 -8 63 403 448 -242 -262 325 192 -90 460 -408 49 -380 168 -402 288 298 92 -348 183 -68 400 -291 305 -340 447 164 -79 25 374 -115 -131 -62 -212 -183 196 27 -6 240 -211 -46 332 -310 74 353 -211 -137 151 -267 15 187 165 267 396 322 427 -304 -14 200 221 -139 -414 -58 -349 -126 260 347 -247 -246 439 394 -439 -377 -415 135 476 226 -150 480 459 -283 167 477 484 415 299 -236 -389 -214 464 -316 499 -450 127 -350 277 239 -151 -118 -155 -212 -223 -94 -89 -138 -106 240 -60 244 220 -101 -186 239 -124 -202 154 -324 62 
500 249766
314 27 -198 313 429 429 -185 -442 168 164 -59 -486 -196 218 272 -433 -68 -334 307 372 -237 379 123 77 118 -500 375 125 28 290 243 -158 169 -455 7 450 475 322 -491 495 338 -50 -139 -6 20 -14 413 -196 152 221 28 415 452 -349 -8 71 3 220 -304 -469 362 -209 225 31 -164 -268 -19 163 -94 -10 -489 96 -208 -128 -410 -336 358 -496 -32 363 -423 -152 -370 -471 352 475 452 -145 195 0 239 409 291 -36 -60 -372 -451 -227 -357 -45 -384 -346 -96 260 27 -6 425 237 350 245 100 -221 -406 -417 -191 446 -442 113 153 -395 -386 -108 -486 -243 -291 306 237 -242 -421 381 -435 -305 -465 -31 456 414 -184 -267 152 18 478 104 298 -76 187 -41 222 245 -428 -124 -298 38 -380 -284 296 181 -478 385 439 -398 266 -143 149 302 -322 457 -432 -6 -310 220 -135 21 177 -337 445 216 122 -480 314 194 248 -484 85 368 -267 233 50 107 118 341 209 237 -302 211 39 229 -480 107 75 -289 180 440 -416 209 -397 -119 -75 225 -99 91 272 -351 108 -143 -130 -307 -58 420 300 -440 -387 362 297 164 73 188 -107 93 148 468 -344 180 409 -260 -111 -488 474 314 -410 -125 258 362 25 -134 71 395 59 -487 -333 -141 -75 -220 73 -277 444 -354 -237 189 -408 411 158 100 91 67 -307 480 431 -333 -353 -479 42 405 -265 -81 123 -342 166 34 23 -315 393 449 318 319 -328 262 465 287 304 -91 199 -38 -490 142 -119 -297 475 -188 -130 -26 186 -236 231 -227 -464 -146 -68 202 388 455 -260 -367 -244 -442 452 -72 -328 -230 68 476 179 -233 -62 -459 -239 319 -256 -264 -16 466 210 22 83 -59 295 119 -353 79 173 -113 35 413 373 291 471 325 72 -356 -53 -8 -28 -21 259 411 20 -480 -418 -383 109 418 -65 -181 -60 -482 -387 88 137 112 -333 163 -500 -446 76 373 -302 400 -450 -378 -104 350 114 368 -171 225 279 -299 -403 -286 170 206 -16 -394 -122 425 -24 -9 365 -34 -397 -468 -371 -45 -61 -443 -172 137 457 231 -389 -147 81 225 -278 410 302 -147 -37 399 419 -14 106 -96 444 -164 181 -80 327 46 386 -218 430 -133 238 -131 -75 -434 -142 -118 297 -31 -412 230 46 162 492 -152 -133 -44 100 -213 -206 58 191 -262 394 224 158 221 -230 397 -145 -300 116 -407 422 -459 12 -220 276 161 250 364 -108 148 26 -116 497 393 -308 97 32 -14 7 75 224 -99 -201 235 -26 421 484 329 -26 100 -225 -104 142 287 176 -82 
500 145800
278 134 -308 -73 12 429 276 -95 -379 225 -210 108 -268 365 184 133 17 -81 -41 -62 -97 140 412 -496 415 160 498 -446 -163 -84 355 -33 -98 -101 -254 414 180 -478 171 302 247 461 262 -169 327 -54 -184 -156 -282 275 -366 473 -84 -453 -171 183 -441 -173 238 -104 -405 445 -284 -3 -156 -38 -89 -475 -15 -65 -321 84 -104 -207 416 75 -409 -268 -229 -191 -140 -94 -217 -372 305 464 311 364 -208 49 113 239 346 329 -263 -309 143 -500 68 -20 -65 -401 -435 -316 -108 -167 259 -17 417 -117 293 -223 289 -72 257 446 244 69 -338 -464 -30 -373 276 -183 456 -135 360 452 217 -72 -68 5 379 -3 189 -229 -318 300 254 -400 -317 399 229 -176 179 -13 -230 -76 -444 285 312 26 412 440 -305 -279 -195 -445 -327 -125 335 105 380 214 -45 69 -163 137 -131 -408 237 405 -157 319 -271 -125 -194 352 -349 214 137 -185 -260 49 -244 288 -378 413 343 -205 -212 179 -247 -332 245 208 237 -417 197 -41 27 287 -136 370 106 445 97 412 297 248 126 -66 -436 -282 -164 172 -494 -42 85 -298 -394 374 381 -141 -106 478 -81 -16 -439 -384 443 88 403 159 311 361 104 -92 -375 254 9 -397 -312 425 174 376 449 -468 187 34 -266 293 260 -33 -348 -345 -54 71 491 7 -461 434 -52 -205 445 -241 -344 401 -481 -218 155 380 -115 -304 -343 -441 -428 -394 -408 -389 493 178 404 -395 -354 -92 112 444 -169 103 -49 -129 389 -249 166 334 10 -326 -412 382 308 243 262 -306 439 -228 105 -136 230 -451 -25 223 228 232 329 -274 -360 293 -330 -28 -103 473 -305 -214 -275 -287 -27 -413 -113 61 469 -304 156 -416 242 96 -144 -153 -188 -414 249 287 -338 329 19 -157 -445 -488 -364 77 -16 385 50 179 -328 275 392 497 363 131 410 -316 327 66 -232 69 14 476 -231 326 -85 -130 466 77 -301 337 420 106 349 -92 -317 185 -206 -415 216 318 -287 460 315 -72 92 225 112 271 143 -267 193 -342 -291 -186 484 124 184 450 53 -265 -360 325 341 489 -266 -124 -473 380 313 243 198 -474 204 365 -46 148 442 -81 271 85 152 -36 243 -287 278 228 -310 314 -470 243 -451 -330 421 -258 12 7 470 39 -113 284 134 -63 310 190 -198 117 -162 -256 -464 -38 -319 -460 426 425 -247 -443 -495 -57 -277 -113 39 -227 58 -40 367 -78 467 338 461 206 122 95 -357 284 -214 297 -99 476 -459 289 -62 74 329 217 -1 -65 -374 4 -270 -151 392 269 122 302 
500 581
-10 -276 400 180 37 458 -198 -16 -399 -62 -378 398 -308 -49 -209 481 389 366 -337 458 217 450 -416 74 -320 -66 318 -198 -92 120 -265 -250 -304 -364 430 -415 -406 84 69 48 -477 -456 298 67 -5 -410 48 236 -192 -437 195 -475 -487 -369 451 46 65 269 348 474 -111 -417 76 437 71 7 -478 -482 -57 444 418 318 488 216 -115 335 -342 -214 71 318 -151 118 -304 215 250 -353 -239 167 417 -391 493 158 44 70 96 -32 -71 470 338 372 -234 256 -457 106 -28 -72 -59 483 214 -135 301 -84 483 497 -369 -415 -355 -108 253 -438 353 -402 -428 -103 168 168 -283 -51 491 -445 174 -243 -337 217 364 488 -355 157 471 -288 -478 272 128 -142 -378 259 -57 -233 -349 48 181 -496 147 -395 253 167 -374 -177 -383 -383 -122 -357 -126 394 360 90 382 -143 -252 353 69 122 477 49 480 -401 -192 276 218 311 -176 251 -333 -177 -291 421 491 -165 96 -40 -48 -174 103 -322 -280 315 269 -398 -328 -131 -193 -406 -157 -215 143 -176 236 -196 -400 307 -385 276 410 135 -400 -381 408 -57 -46 4 -245 -242 330 -290 -63 -449 -475 58 -347 -303 -221 313 143 122 -402 -213 446 -314 91 -102 -155 -442 -325 108 193 127 227 101 -430 34 -395 177 292 436 387 -271 -161 412 -361 -8 -39 -82 157 -395 -459 -393 -108 -161 -206 -165 238 491 245 -235 99 291 -256 179 -108 166 -287 350 -157 -143 -362 -270 87 -23 -358 226 -179 455 -3 479 -440 38 -414 304 377 232 -9 467 -424 -263 84 175 -472 328 206 272 -6 419 474 337 -223 -388 -433 -284 441 61 -206 115 17 291 94 77 -171 -468 234 59 -383 -275 378 -307 -38 -37 -280 342 -209 427 115 286 -302 89 475 -25 -446 -457 -457 495 456 -162 110 -27 -371 56 -97 311 441 -11 -130 -442 214 100 103 -323 -437 323 -129 -145 -398 486 -7 -199 428 -32 276 482 363 -328 -171 320 10 440 145 491 348 -100 302 289 389 24 -301 104 -375 302 -367 40 478 4 395 -420 343 240 -119 271 -291 10 253 -428 182 -418 -256 -308 -126 390 35 -277 142 -162 -136 32 -286 64 -12 191 218 473 -268 196 -23 -373 277 172 368 -342 -57 -71 -480 48 353 202 483 -402 -254 -143 -160 282 432 482 -28 -203 366 186 -287 206 -122 431 179 110 128 9 -411 257 -319 -191 -233 477 238 -212 25 444 490 360 -458 89 -282 234 -129 2 -432 343 -349 -213 381 -136 493 -241 148 -475 -279 128 34 -189 -115 -433 472 -496 -456 211 
500 -201708
422 -493 135 282 -451 -276 -148 -365 447 -293 -297 -358 -142 -10 -477 -425 336 135 -425 361 356 203 -105 19 -60 314 -8 -56 -289 55 -411 -15 62 224 119 463 -200 472 -50 -253 31 153 241 389 -4 264 -184 -168 251 -109 45 460 -54 -208 -21 386 -394 -177 -169 -183 -270 -80 -346 292 -4 274 -393 -352 -402 57 -105 129 63 -12 370 -89 -248 -313 -405 -144 430 140 -184 -123 432 -353 -237 -110 -29 446 208 201 218 -286 346 66 488 -195 214 -414 363 -39 215 278 449 438 -311 54 -23 -216 410 -93 -224 -274 -364 -292 -127 252 98 344 198 158 398 269 373 -404 -165 -287 -99 -450 -200 -236 11 -133 -106 313 -195 83 -133 282 -281 -223 -310 347 355 178 -93 -272 -70 -142 -75 -19 -484 175 -250 -259 271 437 -45 -328 487 -393 -63 351 -26 -317 164 280 -381 -117 62 190 160 -396 38 15 283 445 -405 -287 -197 20 194 172 47 -204 -235 -182 -266 -428 -9 73 -321 -220 276 154 -37 292 286 82 -325 -152 -227 -313 -47 311 202 88 -392 -350 -347 264 -478 348 -64 70 -4 -447 388 230 -374 -121 -196 -195 -489 -68 -189 -25 -275 -403 -91 -100 -54 182 88 -249 345 142 -309 -194 292 -156 -430 -333 44 6 237 -459 59 125 -377 -463 -143 279 -157 -132 -288 -494 -305 -63 -396 105 -311 402 139 277 5 337 272 196 -357 416 -108 65 -417 437 -77 320 478 -166 298 -47 -128 7 -267 -433 -273 -55 -427 275 -266 29 -120 -77 -69 -129 -299 -212 60 -175 -164 -445 -259 229 120 177 166 -457 -3 496 -122 147 -51 250 6 -466 -331 -414 -21 -258 361 213 272 93 -363 55 464 190 344 25 -485 -320 80 108 -239 53 -215 -221 96 -365 -225 -174 -366 -423 -72 141 -389 97 227 443 -308 440 -492 464 33 -355 371 -3 -313 -433 -126 54 100 -193 15 361 360 300 -359 308 -213 268 -365 -78 -303 63 -85 161 13 -6 104 205 -66 112 -479 467 -390 392 316 149 -188 -309 56 412 350 -429 -375 -438 371 118 370 -341 -113 -143 433 436 -227 -152 97 286 -306 -299 -9 128 -334 12 95 128 256 263 129 -432 -194 -315 -168 -344 -244 -42 70 -20 -424 293 491 315 -350 424 252 275 124 -151 61 318 403 -96 298 -79 -232 -107 -451 -475 8 -470 -55 315 -284 278 471 -28 88 42 452 -484 187 443 332 337 -281 84 465 343 -215 378 -487 40 -217 311 461 403 56 -138 -220 -435 -107 226 380 109 356 -297 -67 -56 -403 238 -40 284 181 -208 474 -247 -272 -209 
