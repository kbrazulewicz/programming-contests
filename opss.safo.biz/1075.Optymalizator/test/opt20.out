NIE
NIE
TAK
TAK
NIE
TAK
NIE
NIE
TAK
NIE
